src/renderer/components/ft-input-tags/ft-input-tags.vue

```diff
@@ -21,7 +21,26 @@
       :force-action-button-icon-name="['fas', 'arrow-right']"
       @click="updateTags"
     />
-    <div class="ft-tag-box">
+    <div
+      v-if="tagList.length >= 1"
+      class="checkbox-container"
+    >
+      <input
+        :id="label"
+        type="checkbox"
+        :checked="showTags"
+        @change="toggleShowTags"
+        @keydown.enter.prevent="toggleShowTags"
+        @keydown.space.prevent="toggleShowTags"
```

Comment:
**suggestion:** These can be removed since you're using the base HTML component.